src/shell/commands/cold_backup.cpp

```diff
@@ -143,64 +145,51 @@ bool add_backup_policy(command_executor *e, shell_context *sc, arguments args)
 
 bool ls_backup_policy(command_executor *e, shell_context *sc, arguments args)
 {
-    ::dsn::error_code err = sc->ddl_client->ls_backup_policy();
+    argh::parser cmd(args.argc, args.argv);
+    const bool json = cmd[{"-j", "--json"}];
+
+    ::dsn::error_code err = sc->ddl_client->ls_backup_policy(json);
     if (err != ::dsn::ERR_OK) {
         std::cout << "ls backup policy failed" << std::endl;
-    } else {
-        std::cout << std::endl << "ls backup policy succeed" << std::endl;
     }
     return true;
 }
 
 bool query_backup_policy(command_executor *e, shell_context *sc, arguments args)
 {
-    static struct option long_options[] = {{"policy_name", required_argument, 0, 'p'},
-                                           {"backup_info_cnt", required_argument, 0, 'b'},
-                                           {0, 0, 0, 0}};
-    std::vector<std::string> policy_names;
-    int backup_info_cnt = 3;
+    const std::string query_backup_policy_help = "<policy_name> [-b|--backup_info_cnt] [-j|--json]";
+    argh::parser cmd(args.argc, args.argv);
```

Comment:
Similar to https://github.com/apache/incubator-pegasus/pull/2027, it's needed to add a parameter `PREFER_PARAM_FOR_UNREG_OPTION`. @empiredan right?